apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -538,6 +538,9 @@ AND notEquals(start_time, toDateTime64('1970-01-01 00:00:00.000', 9)),
                 <if(sort_has_feedback_scores)>
                 LEFT JOIN feedback_scores_agg fsagg ON fsagg.entity_id = t.id
                 <endif>
+                <if(sort_has_span_statistics)>
+                LEFT JOIN spans_agg s ON t.id = s.trace_id
```

Comment:
Just make sure to test such a query for big workspaces, by bringing the join back to this level, we are defeating the purpose of applying a limit before joining. This was done for feedback scores, as the table itself was very small, but the traces one is the most significant to join with. 


It will probably be fine for most small workspaces, but the issue is for larger ones (FYI, the sorting function is limited to workspaces with 50GB, so we only need to try with workspaces closer to this number).